src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  // here we filter males for specific century, either we just return males.
+  const male = people.filter(person => !century
```

Comment:
maybe it is easier to read when condition parameter specifies like truly from start (i mean person => century)